atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitions.java

```diff
@@ -216,9 +224,45 @@ static boolean isMatchingCf(CfDef clientSide, CfDef clusterSide) {
     private static void logMismatch(String fieldName, String tableName,
             Object clientSideVersion, Object clusterSideVersion) {
         log.info("Found client/server disagreement on {} for table {}. (client = ({}), server = ({}))",
-                fieldName,
-                tableName,
-                clientSideVersion,
-                clusterSideVersion);
+                SafeArg.of("disagreementType", fieldName),
+                UnsafeArg.of("table", tableName),
```

Comment:
`LoggingArgs.tableRef()` but that works only for refs I think, maybe there is something for tablenames or we can log `TableRef.createUnsafe(name)` . This will result in greater visibility.